Source/Peripheral.swift

```diff
@@ -347,6 +347,19 @@ public class Peripheral {
         return ensureValidPeripheralState(for: observable)
     }
 
+    public func observeCharacteristicStateChanged(for characteristic: Characteristic) -> Observable<Characteristic> {
```

Comment:
You didn't add comment with documentation to this method

Reply:
I think name of this method is not good. We don't need characteristic part - we know this from parameters. This is not also changing characteristic state. In [documentation](https://developer.apple.com/documentation/corebluetooth/cbperipheral/1518949-setnotifyvalue) there is not information about characteristic state, it is notifying or not notifying, so maybe we can change this method name to `observeNotifyValue`